### PR TITLE
Editorial: add note on file extension

### LINKED ISSUE #689: manifest.webmanifest or manifest.json
I notice the w3c spec mentions `manifest.webmanifest` yet I see Google and Firefox and other browsers say to use `manifest.json`

Simple question: Should we use both or just pick one? ~As I can't find the answer in the spec. Plus the spec doesn't mention `manifest.json` either which seems strange.

(I been using the `.json` and seen no problems for several years, just wondered if I should add both).

Thanks.

### PR DESCRIPTION
**Pull request by christianliebel**
Closes #689

This change (choose one):

* [ ] Breaks existing normative behavior (please add label "breaking")
* [ ] Adds new normative requirements
* [ ] Adds new normative recommendations or optional items
* [x] Makes only editorial changes (only changes informative sections, or
  changes normative sections without changing behavior)
* [ ] Is a "chore" (metadata, formatting, fixing warnings, etc).

Commit message:

Editorial: add note on file extension

***
<a href="https://pr-preview.s3.amazonaws.com/christianliebel/manifest/pull/802.html" title="Last updated on Sep 24, 2019, 5:35 AM UTC (3f8dfe2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/manifest/802/ad4e579...christianliebel:3f8dfe2.html" title="Last updated on Sep 24, 2019, 5:35 AM UTC (3f8dfe2)">Diff</a>

### CODE REVIEW

**Comment by mgiuca** [comment]
I can't figure out how to review this... LGTM.

File: index.html [hunk 1]
```diff
@@ -257,6 +257,15 @@ <h3>
           &lt;link rel="icon" sizes="16x16 32x32 48x48" href="lo_def.ico"&gt;
           &lt;link rel="icon" sizes="512x512" href="hi_def.png"&gt;
         </pre>
+        <div class="note" title="manifest.webmanifest or manifest.json?">
+          The official file extension for the manifest is
+          <code>.webmanifest</code>. Some web servers recognize this extension
+          and transfer the file using the official <a>media type for a
```
Comment (marcoscaceres) [inline index.html]:
```suggestion
          and transfer the file using the standardized <a>media type for a
```

File: index.html [hunk 2]
```diff
@@ -257,6 +257,15 @@ <h3>
           &lt;link rel="icon" sizes="16x16 32x32 48x48" href="lo_def.ico"&gt;
           &lt;link rel="icon" sizes="512x512" href="hi_def.png"&gt;
         </pre>
+        <div class="note" title="manifest.webmanifest or manifest.json?">
+          The official file extension for the manifest is
+          <code>.webmanifest</code>. Some web servers recognize this extension
```
Comment (marcoscaceres) [inline index.html]:
```suggestion
          `.webmanifest`. Some web servers recognize this extension
```

File: index.html [hunk 3]
```diff
@@ -257,6 +257,15 @@ <h3>
...
+          and transfer the file using the official <a>media type for a
+          manifest</a> (<code>application/manifest+json</code>). Developers can
```
Comment (marcoscaceres) [inline index.html]:
```suggestion
          manifest</a> (`application/manifest+json`). Developers can
```

File: index.html [hunk 4]
```diff
@@ -257,6 +257,15 @@ <h3>
...
+          also choose a different extension (e.g. <code>.json</code>) or none
```
Comment (marcoscaceres) [inline index.html]:
```suggestion
          also choose a different extension (e.g. `.json`) or none
```

File: index.html [hunk 5]
```diff
@@ -257,6 +257,15 @@ <h3>
...
+          at all (e.g. <code>/api/GetManifest</code>), but must make sure
```
Comment (marcoscaceres) [inline index.html]:
```suggestion
          at all (e.g. `/api/GetManifest`), but are strongly encouraged
```

File: index.html [hunk 6]
```diff
@@ -257,6 +257,15 @@ <h3>
...
+          that the manifest is transferred using the official media type.
```
Comment (marcoscaceres) [inline index.html]:
```suggestion
          that the manifest is transferred using the `application/manifest+json` media type.
```

**Comment by marcoscaceres** [comment]
Thanks @christianliebel!

**Pull request merged**